src/carplay.cs

```diff
@@ -796,6 +815,51 @@ interface CPImageSet : NSSecureCoding {
 		UIImage DarkContentImage { get; }
 	}
 
+	interface ICPTemplateApplicationSceneDelegate { }
+
+	[NoWatch, NoTV, NoMac, iOS (13,0)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject))]
+	interface CPTemplateApplicationSceneDelegate : UISceneDelegate
+	{
+		[Export ("templateApplicationScene:didConnectInterfaceController:toWindow:")]
+		void DidConnect (CPTemplateApplicationScene templateApplicationScene, CPInterfaceController interfaceController, CPWindow window);
+
+		[Export ("templateApplicationScene:didDisconnectInterfaceController:fromWindow:")]
+		void DidDisconnect (CPTemplateApplicationScene templateApplicationScene, CPInterfaceController interfaceController, CPWindow window);
+
+		[Export ("templateApplicationScene:didSelectNavigationAlert:")]
+		void DidSelect (CPTemplateApplicationScene templateApplicationScene, CPNavigationAlert navigationAlert);
+
+		[Export ("templateApplicationScene:didSelectManeuver:")]
+		void DidSelect (CPTemplateApplicationScene templateApplicationScene, CPManeuver maneuver);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (13,0)]
+	[BaseType (typeof (UIScene))]
+	interface CPTemplateApplicationScene
+	{
+		[Export ("initWithSession:connectionOptions:")]
+		IntPtr Constructor (UISceneSession session, UISceneConnectionOptions connectionOptions);
+
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		ICPTemplateApplicationSceneDelegate Delegate { get; set; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Strong)]
+		NSObject WeakDelegate { get; set; }
+
+		[Export ("interfaceController", ArgumentSemantic.Strong)]
+		CPInterfaceController InterfaceController { get; }
+
+		[Export ("carWindow", ArgumentSemantic.Strong)]
+		CPWindow CarWindow { get; }
+
+		[iOS (13, 0)]
```

Comment:
minor (don't update the PR only for that, unless other changes are needed - it can be changed in future betas)

There's no need for an `[iOS (13,0)]` here since it's already on the type-level (so it's inherited, by defaults, on all members - unless they specify something else)